package/tyrquake/tyr_quake.mk

```diff
@@ -0,0 +1,26 @@
+################################################################################
+#
+# lbreakout2
```

Comment:
Slight nitpick: Fix this comment to say `tyrquake`.

Is there any particular reason for naming these files `tyr_quake.{mk,hash}` with the underscore?  I guess it's fine if it still works.

Reply:
I was just up really late and wasn't paying attention.  I'll fix it.